### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by evaveskova**
What was done in this branch:

- Prepared the directory structure
- Implemented the App component
- Implemented the BooksList component
- Implemented the BooksForm component
- Defined the Redux actions and reducers
- Initialized the Redux store

### CODE REVIEW

File: src/containers/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <div>
+    <table>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Title</th>
+          <th>Category</th>
+        </tr>
+      </thead>
+      {books.map(book => (
```
Comment (bellom) [inline src/containers/BooksList.js:16]: We need to import `Book Component`.
`BookList` should displays books list in html table using `Book component`

Example:
`{books.map(book => (<Book book={book} key={book.id} />))}`

File: src/reducers/books.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const bookReducer = (state = null, action) => {
```
Comment (bellom) [inline src/reducers/books.js]: Now!, we need to combine this `book reducers` inside `src/reducers/index.js`
image

**Read the documentation!** 

Example: 
```
combineReducers({
  books,
});
```

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/containers/BooksList.js (hunk 1 above)
Comment (noomdalv) [reply src/containers/BooksList.js:16]: Changes applied,
Book component is now imported into BooksList component and rendering properly.

File: src/reducers/books.js (hunk 2 above)
Comment (noomdalv) [reply src/reducers/books.js]: Corrections done,
reducers/index.js uses now combineReducers method to add books.js reducer

**Comment by noomdalv** [comment]
Corrections completed:

- [x] Book component is now imported into BooksList component and rendering properly.
- [x] reducers/index.js uses now combineReducers method to add books.js reducer.

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

NOTE: Config linter's check in the next milestone on your Github page.

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**